### PR TITLE
Font viewer updates

### PR DESCRIPTION
**Pull request by flukeout**
**Changes**

* Now had light & dark theme
* Code samples for changing font style via HTML or CSS
* Added "Pretty" font name for use in CSS code sample
* Localization strings

Not sure what to do with the localization for "The quick brown fox" - leave as is?

### CODE REVIEW

File: src/extensions/extra/brackets-fonts-viewer/main.js [hunk 1]
```diff
@@ -43,6 +44,13 @@ define(function (require, exports, module) {
     /** @type {Array.<FontView>} */
     var _viewers = {};
 
+    // Get a name for this font for use in the CSS sample code
+    function _getFontName(file) {
+        var filename = file.name.substring(0, file.name.indexOf("."))
```
Comment (humphd) [inline src/extensions/extra/brackets-fonts-viewer/main.js]: Use the Filer Path module for this stuff vs. doing it by hand, probably you want `Path.basename(file.name)`

**Review by humphd — CHANGES_REQUESTED** [review]

File: src/extensions/extra/brackets-fonts-viewer/main.js (hunk 1 above)
Comment (flukeout) [reply src/extensions/extra/brackets-fonts-viewer/main.js]: ``Path.basename(file.name)`` returns ``NunitoSans-LightItalic.ttf``

File: locales/en-US/editor.properties [hunk 2]
```diff
@@ -155,6 +154,16 @@ IMAGE_FILTER_GLOWING_SUN=Glowing Sun
...
+# Font Viewer
+FONT_FILE=Font File
+USE_FONT_INSTRUCTIONS=To use this font, copy and paste this code into your CSS file.
+FONT_TYPE_NORMAL=Normal
+FONT_TYPE_BOLD=Bold
+FONT_TYPE_ITALIC=Italic
+FONT_BOLD_HTML_TAG=Put text inside <code>&lt;strong&gt;</code><code>&lt;/strong&gt;</code> tag.
+FONT_BOLD_CSS_RULE=Add a <code>font-weight: bold;</code> rule.
+FONT_ITALIC_HTML_TAG=Put text inside <code>&lt;em&gt;</code><code>&lt;/em&gt;</code> tag.
+FONT_ITALIC_CSS_RULE=Add a <code>font-style: italic;</code> rule.
```
Comment (humphd) [inline locales/en-US/editor.properties:166]: This seems right, we just need to add them in Thimble.  Get Gideon to OK it, though.

File: src/extensions/extra/brackets-fonts-viewer/main.js (hunk 1 above)
Comment (humphd) [reply src/extensions/extra/brackets-fonts-viewer/main.js]: Ugh, right.  You could do:

```js
var nameWithoutExtension = Path.basename(file.name).replace(Path.extname(file.name), "");
```

**Review by humphd — CHANGES_REQUESTED** [review]
Couple small things, looks good otherwise.

**Pull request merged**